x-pack/legacy/plugins/uptime/server/lib/adapters/monitors/elasticsearch_monitors_adapter.ts

```diff
@@ -69,6 +69,7 @@ export class ElasticsearchMonitorsAdapter implements UMMonitorsAdapter {
             filter: [
               { range: { '@timestamp': { gte: dateRangeStart, lte: dateRangeEnd } } },
               { term: { 'monitor.id': monitorId } },
+              { term: { 'monitor.status': 'up' } },
```

Comment:
So now we are just getting data only for up monitors?

Reply:
Yes exactly - when we designed this feature we realized that there's not much point in displaying the duration for a monitor that is down. From the issue:

> We feel that showing gaps in the chart will let users measure only the performance of the monitor's location while it is up, with the understanding that we drop any down state on the floor to signal that the duration is useless for those checks because a nonfunctioning state leads to unpredictable duration values